src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SpecifyCultureForToLowerAndToUpper.cs

```diff
@@ -51,7 +51,8 @@ protected override void InitializeWorker(CompilationStartAnalysisContext context
 
         private static bool IsToLowerOrToUpper(string methodName)
         {
-            return methodName == ToLowerMethodName || methodName == ToUpperMethodName;
+            // TODO: Compare symbols instead of method name.
```

Comment:
File a tracking issue?

Reply:
I got a PR for it already https://github.com/dotnet/roslyn-analyzers/pull/6382